src/supertux/savegame.hpp

```diff
@@ -65,6 +66,24 @@ struct WorldmapState
 
 class Savegame final
 {
+public:
+  struct Progress
+  {
+  public:
+    Progress() :
+      solved(),
+      total()
+    {}
+    Progress(int solved_, int total_) :
+      solved(solved_),
+      total(total_)
+    {}
+    uint32_t solved;
```

Comment:
m_solved
m_total

Reply:
I didn't think an `m_` prefix is needed here, since this struct is only used to store 2 variables together and has just one simple function to calculate percentage using them.